### DISCUSSION TITLE
Use uupdump version with Memory’s Tech Tips’ Unattended Windows Installation

### DISCUSSION THREAD

**Discussion by 7HRAMZI** [memstechtips/UnattendedWinstall#102] [Q&A]
**Hello, I have something on my mind and I hope you can answer it if possible**
I am thinking of
1\Download a copy of Windows 11 or 10 from the website [UUP dump] 
2\Activate all the options in the [Conversion options] section for [To get a ISO version that contains]
A\Include updates (Windows converter only)
B\Run component cleanup (Windows converter only)
C\Integrate .NET Framework 3.5 (Windows converter only)
D\Use solid (ESD) compression
3\Add file [Memory’s Tech Tips’ Unattended Windows Installation
autounattend]
4\Install the operating system
Q1\The question is, do you recommend that I do this? [Yes or No]And why?
Q2\What do you suggest for me?And why?
**Thanks and sorry for the long question.**
**I wish you success**

**Comment by memstechtips** [comment]
Hi there,

To be quite honest, I haven't ever tried the answer files on the ISO's downloaded from UUP dump.
I do think, if it's something you want to experiment with, go ahead and do it. 😀

You're welcome to report back here if you want.
